curvefs/src/client/fuse_volume_client.cpp

```diff
@@ -306,5 +280,13 @@ void FuseVolumeClient::FlushData() {
     // TODO(xuchaojie) : flush volume data
 }
 
+void FuseVolumeClient::SetSpaceManagerForTesting(SpaceManager *manager) {
```

Comment:
Implement FuseOpFlush. If cto, persist metadata and data, otherwise only persist data

Reply:
fix